hexa/ui/templates/ui/datacard/property_text.html

```diff
@@ -1,11 +1,30 @@
 {% extends "ui/datacard/property.html" %}
 
 {% block value %}
-    {% if markdown %}
-        <div class="prose prose-sm" {% if not translate %}translate="no"{% endif %}>
-            {{ text|default:"-" }}
-        </div>
+{% if markdown %}
+<div class="prose prose-sm" {% if not translate %}translate="no" {% endif %}>
+    {{ text|default:"-" }}
+</div>
+{% else %}
+<div x-data="SecretSection()">
```

Comment:
It's more `SecretProperty` than `SecretSection`

Reply:
I think it's better to not instantiate the `SecretSection` if the property is not a secret one. (So only called `SecretSection` if `{%secret%}`

Reply:
Okay, that's right, will update it.